internal/config/config.go

```diff
@@ -215,9 +215,21 @@ type Instrumentation struct {
 
 // SmartRetry represents the settings for retry strategy.
 type SmartRetry struct {
+	FailedOnly bool `yaml:"failedOnly" json:"-"`
+	// FailedClassesOnly was introduced as the first iteration of smart retry
+	// and was applicable to mobile only.
+	// DEPRECATED. Use FailedOnly instead.
 	FailedClassesOnly bool `yaml:"failedClassesOnly" json:"-"`
```

Comment:
We should spit out a warning if `FailedClassesOnly` is set, so that users migrate over to `FailedOnly`.

Reply:
sure!